### PR TITLE
[TD-4228]<fix>: fix install error when grafana submodule not ready

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: packaging/deb/makedeb.sh [hunk 1]
```diff
@@ -58,7 +58,9 @@ cp ${compile_dir}/build/lib/${libfile}              ${pkg_dir}${install_home_pat
 cp ${compile_dir}/../src/inc/taos.h                 ${pkg_dir}${install_home_path}/include
 cp ${compile_dir}/../src/inc/taoserror.h            ${pkg_dir}${install_home_path}/include
 cp -r ${top_dir}/tests/examples/*                   ${pkg_dir}${install_home_path}/examples
-cp -r ${top_dir}/src/connector/grafanaplugin/dist   ${pkg_dir}${install_home_path}/connector/grafanaplugin
+if [ -d "${top_dir}/src/connector/grafanaplugin/dist" ]; then
```
Comment (sangshuduo) [inline packaging/deb/makedeb.sh:61]: it may hide potential mistake if the submodule not updated

**Review by sangshuduo — CHANGES_REQUESTED** [review]
the user who build package should take care all submodule be updated. otherwise the process should be terminated with a clear error msg.

File: packaging/deb/makedeb.sh (hunk 1 above)
Comment (zitsen) [reply packaging/deb/makedeb.sh:61]: Added error in packaging, and warning in installing scripts.

**Review by zitsen — COMMENTED** [review]
All things done.

**Review by sangshuduo — APPROVED** [review]

**Pull request merged**